compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3952,31 +3952,39 @@ genCompareAndBranchInstructionIfPossible(TR::CodeGenerator * cg, TR::Node * node
    return cursor;
    }
 
-TR::InstOpCode::S390BranchCondition useBranchOnZeroConditionMaskIfPossible(TR::Node * node, TR::InstOpCode::S390BranchCondition brCond)
+/**
+ * \fn convertComparisonBranchConditionToTestUnderMaskBranchCondition(TR::Node * node, TR::InstOpCode::S390BranchCondition brCond)
```

Comment:
I don't think this is necessary. Doxygen will automatically group this function comment with the subsequent definition.